client/src/Molecules.elm

```diff
@@ -564,8 +573,6 @@ levelNEncoder level1 =
         , ( "level"
           , Encode.int 3
           )
-        , ( "inserts", Encode.list (\insert -> level0Encoder insert) level1.inserts )
-        , ( "backbone", backboneEncoder (Maybe.withDefault initBackbone level1.backbone) )
```

Comment:
Indeed, but you do need a `children` key?

Reply:
I don't really follow... schemas.py: VectorIn has a field `children`.
So I assume I need to pass it to the server.

Reply:
Indeed, but there is no `children` key in your encoder.

Reply:
What I meant to say was that `levelNEncoder` does have a `children` key on line 580:

```elm
         ( "children"
          , Encode.list Encode.int
                ((toList level1.backbone
                    |> List.map .id
                 )
                    ++ List.map .id level1.inserts
                )
          )

```

Is this OK? Or do you mean something else?

Reply:
Indeed. I missed this. Thanks